src/validation/custom.validation.ts

```diff
@@ -41,6 +41,16 @@ export const checkDateIsInPast = (errMsg: string, day: string = "", month: strin
   return true;
 };
 
+export const checkDateIsInPastOrToday = (errMsg: string, day: string = "", month: string = "", year: string = "") => {
```

Comment:
Can we have `checkDateIsInPastOrToday` instead of `checkDateIsInPast`? and change the message to include `today`?

Reply:
Not at the moment. We have other screens that don't allow today, although I have raised this as something we might want to change. So we still need both at the moment.

Reply:
You are right but it is likely that we will have both error messages in the Error list, one that says `The date must be in the past` and `The date must be today or in the past`. 

It is an edge case and for MVP I will approve. Will need an update, can be done later.